src/test/groovy/graphql/analysis/QueryTraverserTest.groovy

```diff
@@ -581,6 +581,62 @@ class QueryTraverserTest extends Specification {
         'preOrder'  | 'visitPreOrder'
     }
 
+    @Unroll
+    def "traverse a query when a default variable is a list: (#order)"() {
+        given:
+        def schema = TestUtil.schema("""
+            type Query{
+                foo(arg1: [String]): String
+            }
+        """)
+        def visitor = mockQueryVisitor()
+        def query = createQuery("""
+            query myQuery(\$myVar: [String] = ["hello default"]) {foo(arg1: \$myVar)} 
+            """)
```

Comment:
if you use Groovy ''' triple quotes then all the " quotes inside do NOT need quoting

I tened to use ''' quotes for all things graphql